local_tests/test_models.py

```diff
@@ -81,13 +95,23 @@ def __init__(self, command, name=None, output=None, desc=None):
         super().__init__(name=name, output=output, desc=desc)
         
     def execute(self):
+        stderr_file = open(stderr_file_name, 'w+')
         result = subprocess.run(
             self.command.split(), 
             stdout=subprocess.PIPE,
             # @TODO: Collect stderr somewhere. In the event of an error, point user to the error file.
-            stderr=FNULL 
+            stderr=stderr_file 
         )
-        return result.stdout.decode()
+        stderr_file.seek(0) # go back to beginning of file so we can read the whole thing
+        stderr_str = stderr_file.read()
+        # Remove temp file.
```

Comment:
Hm is there really no way to capture the stderr into a variable without creating a temp file?

Reply:
I tried using `io.StringIO` which would write directly to the buffer but it doesn't work because `subprocess` spawns a whole separate process that writes to a file by its file number and you have to actually open a file for it to have a file number. So, no